src/core/src/package_managers/YumPackageManager.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
 
         # if an Auto Patching request comes in on a CentOS machine with Security and/or Critical classifications selected, we need to install all patches
         installation_included_classifications = [] if execution_config.included_classifications_list is None else execution_config.included_classifications_list
-        if execution_config.maintenance_run_id is not None and execution_config.operation.lower() == Constants.INSTALLATION.lower() \
+        if execution_config.health_store_id is not str() and execution_config.operation.lower() == Constants.INSTALLATION.lower() \
```

Comment:
@kjohn-msft, I've used healthstoreid for auto patch validation instead of maxpatchpublishdate since the presence of healthstoreid indicates an AutoPatch request vs maxpatchpublishdate can also come in as an API input